### PR TITLE
Secure CPU Type concept feature page

### PR DESCRIPTION
**Pull request by ljelinkova**
New feature page for Secure CPU Type concept.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @ljelinkova

This pull request needs review by: @michalskrivanek @evol262

### CODE REVIEW

File: source/develop/release-management/features/virt/secure-cpus.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Improved maintainability of supported CPU Types.
+
+## User Experience
+
+If the CPU configuration is different in the cluster table than in the vdc_options table, it is important to notify the user. The following warnings are displayed in the UI:
+
+- cluster list - a warning that the cluster contains host(s) that are not fully compatible with the current CPU Type configuration
+- host list and detail - a warning that the host is not fully compatible with the current CPU Type configuration
+- vm list and detail - a warning that the VM CPU configuration is not compatible with the current CPU Type configuration
```
Comment (michalskrivanek) [inline source/develop/release-management/features/virt/secure-cpus.md]: some more details and an example picture would be nice

File: source/develop/release-management/features/virt/secure-cpus.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
```
Comment (doron-fediuck) [inline source/develop/release-management/features/virt/secure-cpus.md]: Can you also provide details of REST API changes?
ie- how do I know what is secure and what is default using the API? etc.

File: source/develop/release-management/features/virt/secure-cpus.md (hunk 1 above)
Comment (ljelinkova) [reply source/develop/release-management/features/virt/secure-cpus.md]: Added some more details and example pictures.

File: source/develop/release-management/features/virt/secure-cpus.md (hunk 2 above)
Comment (ljelinkova) [reply source/develop/release-management/features/virt/secure-cpus.md]: This feature should not affect the REST API other then that CPU Types are renamed but I've stated it there explicitly.

**Comment by michalskrivanek** [comment]
will need a followup for Kernel Features in Host tab enhancements

**Pull request merged**